templates/metal/interrupt_handlers.h.j2

```diff
@@ -21,7 +21,7 @@
 {% endif %}
 
 {% if global_interrupts is defined %}
-#define __METAL_NUM_GLOBAL_INTERRUPTS {{ global_interrupts.irqs|length }}
+#define __METAL_NUM_GLOBAL_INTERRUPTS {{ global_interrupts.irqs|length + 1 }}
```

Comment:
I think this should probably have the correct number and fix up the allocation in the array instead